tools/proofers/proof_frame_enh.inc

```diff
@@ -6,7 +6,7 @@ include_once('PPage.inc');
 include_once('image_block_enh.inc');
 include_once('button_menu.inc');
 include_once('preview.inc');
-include_once('codepoint_validator.inc');
+include_once($relPath.'codepoint_validator.inc');
```

Comment:
Move this up before `PPage.inc` so it's with the other files that are in `pinc/`.

Reply:
Done this.